index.js

```diff
@@ -23,10 +23,23 @@ function handleRelativePath(_relativePath) {
 }
 
 module.exports = walkSync;
-function walkSync (baseDir, _options, _relativePath) {
+function walkSync(baseDir, _options) {
+  var options = handleOptions(_options);
+
+  return _walkSync(baseDir, options).map(function(entry) {
+    return entry.relativePath;
+  });
```

Comment:
perf is not within 5% or so before, FS noise seems to make it tricky to be sure. But the speed should now be reasonable